src/net_processing.cpp

```diff
@@ -224,9 +224,23 @@ struct Peer {
 
     /** A vector of addresses to send to the peer, limited to MAX_ADDR_TO_SEND. */
     std::vector<CAddress> m_addrs_to_send;
-    /** Probabilistic filter of addresses that this peer already knows.
-     *  Used to avoid relaying addresses to this peer more than once. */
-    const std::unique_ptr<CRollingBloomFilter> m_addr_known;
+    /** Bloom filter to track recent addr messages relayed with this peer.
+     *
+     *  We initialize this filter for outbound peers (other than
+     *  block-relay-only connections) or when an inbound peer sends us an
+     *  address related message (ADDR, ADDRV2, GETADDR).
+     *
+     *  We use the presence of this filter to decide whether a peer is eligible
+     *  for trickle relay of addr messages. This avoids relaying to peers that
```

Comment:
I'd very much prefer not to introduce the word 'trickle' here. I've only ever heard the word 'trickle' being used to refer to transaction propagation (eg #7125). Adding the word here would be overloading the term with a new meaning.

Reply:
ah gotcha. I updated the language to use "gossiping addr messages", does that feel more appropriate / consistent?

Reply:
"gossiping addr messages" SGTM!